src/trader.js

```diff
@@ -89,13 +94,24 @@ socket.on("message", (message) => {
     console.log(colors.magenta("NBT Message: " + message))
 })
 
-socket.on("buy_signal", async (signal) => {
+socket.on("connect_error", (error) => {
```

Comment:
Please create a separate PR for those changes too. When doing so, please have all `socket.on`s first, then their actual functions.